### PR TITLE
Sojutest

### PR DESCRIPTION
**Pull request by soju83191**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -32,8 +32,25 @@
   appearance: checkbox;
 }
 
+.customfont1{
```
Comment (jessanc) [inline app/assets/stylesheets/application.scss]: call this something else, custom font1 is too generic

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -32,8 +32,25 @@
...
+    font-family: Montserrat, Verdana, Geneva, sans-serif;
+    letter-spacing: -0.3px;
+    font-weight: 400;
+    color: #ffffff;
+    margin: 0;
+    text-align: center;
+}
+.customfont2{
```
Comment (jessanc) [inline app/assets/stylesheets/application.scss]: same as last comment

File: app/views/static_pages/homepage.html.erb [hunk 3]
```diff
@@ -1,31 +1,87 @@
-<div class="jumbotron" style="min-height: 460px; background-image: url(assets/healthy.jpg);">
+<div class="jumbotron" style="min-height: 800px; background-image: url(assets/cover3.jpg); background-size: cover; background-position: center">
```
Comment (jessanc) [inline app/views/static_pages/homepage.html.erb]: rename file to something else, background-main-page.jpg?

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -15,36 +15,100 @@
...
-  height: 25px;
-  width: 25px;
-  -webkit-appearance: checkbox;
-  -moz-appearance: checkbox;
-  -ms-appearance: checkbox;
-  -o-appearance: checkbox;
-  appearance: checkbox;
+
+ body {
+   font-size: 1em !important;
```
Comment (soju83191) [inline app/assets/stylesheets/application.scss]: remove this shi

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -15,34 +15,56 @@
...
-    letter-spacing: -0.3px;
-    font-weight: 400;
+ body {
+   font-size: 1em !important;
+   font-weight: 700;
+   font-family: 'Ubuntu', sans-serif;
+ }
+
+
+ // font-family: 'Amiri', serif;
```
Comment (jessanc) [inline app/assets/stylesheets/application.scss:34]: remove if not used

File: app/assets/stylesheets/application.scss [hunk 6]
```diff
@@ -15,34 +15,56 @@
...
+ // font-family: 'Sulphur Point', sans-serif;
+ // font-family: 'Fjord One', serif;
+ // font-family: 'IM Fell DW Pica', serif;
+ .flex-container {
+   display: flex;
+   flex-wrap: nowrap;
+   background-color: white;
+ }
+
+ .flex-container > div {
```
Comment (jessanc) [inline app/assets/stylesheets/application.scss:45]: what does > this mean?

File: app/assets/stylesheets/application.scss [hunk 7]
```diff
@@ -15,34 +15,56 @@
...
+   background-color: #ffffff;
+   margin: 10px;
+   text-align: center;
+   line-height: 75px;
+ }
+
+.titlefont{
+    font-family: 'Sen', sans-serif;
+    letter-spacing: 0.4px;
+    word-spacing: 8px;
```
Comment (jessanc) [inline app/assets/stylesheets/application.scss:55]: shouldn't we use em? so. its relevant to screen size?

File: app/assets/stylesheets/application.scss [hunk 8]
```diff
@@ -15,34 +15,56 @@
...
+ .flex-container > div {
+   background-color: #ffffff;
+   margin: 10px;
+   text-align: center;
+   line-height: 75px;
+ }
+
+.titlefont{
+    font-family: 'Sen', sans-serif;
+    letter-spacing: 0.4px;
```
Comment (jessanc) [inline app/assets/stylesheets/application.scss:54]: em?

File: app/assets/stylesheets/application.scss [hunk 9]
```diff
@@ -15,34 +15,56 @@
...
     text-align: center;
+    line-height: 1.6;
 }
-.customfont2{
-    font-family: Times New Roman;
+
+
+.mpfonttitle{
+    font-family: 'Sen', sans-serif;
     letter-spacing: -0.3px;
```
Comment (jessanc) [inline app/assets/stylesheets/application.scss:66]: em based?

File: app/assets/stylesheets/application.scss [hunk 10]
```diff
@@ -52,8 +74,35 @@
  justify-content: space-between;
 }
 
- .nav-item {
-   align-items: center;
+.mpfontheading{
+    font-family: 'Sen', sans-serif;
+    letter-spacing: -0.3px;
```
Comment (jessanc) [inline app/assets/stylesheets/application.scss:79]: em needed?

File: app/assets/stylesheets/application.scss (hunk 7 above)
Comment (jessanc) [reply app/assets/stylesheets/application.scss:55]: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px guide on what to use.

**Pull request merged**